package.json

```diff
@@ -29,8 +29,8 @@
     "lib": "lib"
   },
   "dependencies": {
-    "readable-stream": "~1.0.2",
-    "semver": "~4.3.3"
+    "readable-stream": "~2.0.5",
+    "semver": "~5.1.0"
```

Comment:
> "semver": "~4.3.3" -> "~5.1.0"

Includes [a breaking change](https://github.com/npm/node-semver/commit/41b56fa573d1ef0d1718777a8a21ae1b57ebcd83), but it doesn't affect log4js.